homeassistant/components/point/alarm_control_panel.py

```diff
@@ -0,0 +1,84 @@
+"""
+Support for Minut Point.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/point/
```

Comment:
We're removing the url from the module docstring. Just keep the very first line.